releases/client/settings

```diff
@@ -1,3 +1,4 @@
+FOREMAN_VERSION=$VERSION
```

Comment:
This line now exists twice in this file

Reply:
Snuck right in there! Also reading comments out of order is both confusing and fun.